pkg/swagger/typewalker.go

```diff
@@ -183,7 +183,7 @@ func (tw *typeWalker) define(definitions Definitions, name string) {
 }
 
 // define adds a Definition for the named types in the given package
-func define(definitions Definitions, pkgname string, xmsEnumList map[string]struct{}, names ...string) error {
+func define(definitions Definitions, pkgname string, xmsEnumList map[string]struct{}, names []string) error {
```

Comment:
could keep this the same and write `err := define(s.Definitions, api, xmsEnumList, names...)` above